docs/docs/js/index.html

```diff
@@ -983,22 +983,22 @@ <h2 id="sample-api-usage">Sample API Usage</h2>
 
 <p>Given <code class="highlighter-rouge">demo.was</code> (encoded to <code class="highlighter-rouge">demo.wasm</code>):</p>
 
-<pre><code class="language-lisp">(module
-    (import "js" "import1" (func $i1))
-    (import "js" "import2" (func $i2))
-    (func $main (call $i1))
-    (start $main)
-    (func (export "f") (call $i2))
-)
-</code></pre>
+<div class="language-lisp highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="p">(</span><span class="nv">module</span>
```

Comment:
Can you update all of this in a separate PR?

Reply:
I guess yeah, this folder is supposed to go away altogether, this change was triggered by dependency update.

Reply:
I've left current contents of `docs` folder in `gh-pages` branch and switched production to use that as source of truth; now I've removed `docs` folder in this PR, so this change is no longer relevant.

Can you please take another look overall?